### PR TITLE
fix(notification)btn

### PR DESCRIPTION
**Pull request by SOAP1914**

### CODE REVIEW

**Comment by SOAP1914** [comment]
@clarkjefcoat @wski or @arw180

**Comment by wski** [comment]
:+1:

File: app/js/components/notification/UserNotificationDropdown.jsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ var UserNotificationDropdown = React.createClass({
             <li data-toggle="tooltip" data-placement="bottom" data-original-title={tooltip} className={notificationButtonClasses} id="notification-dropdown">
                 <a href="#" data-toggle="dropdown" id="tourstop-notifications">
                     <i className={bellClassNames}></i>
+                    <span className="hidden-span"> notifications {this.state.notifications.length} unread notifications</span>
```
Comment (wski) [inline app/js/components/notification/UserNotificationDropdown.jsx]: `{(this.state.notifications) ? this.state.notifications.length : 0}`

**Comment by clarkjefcoat** [comment]
So if you have 2 notifications this will says "notifications 2 unread notifications".  Is that what you want?

**Pull request closed without merging**